packages/util/src/bytes.ts

```diff
@@ -1,16 +1,35 @@
 import { getRandomBytesSync } from 'ethereum-cryptography/random.js'
-import { bytesToHex, bytesToUtf8, hexToBytes } from 'ethereum-cryptography/utils.js'
+// eslint-disable-next-line no-restricted-imports
+import {
+  bytesToHex as _bytesToUnprefixedHex,
+  hexToBytes as _unprefixedHexToBytes,
+} from 'ethereum-cryptography/utils.js'
+
+// Note: ethereum-cryptography/hexToBytes is imported as "unprefixedHexToBytes"
+// This name is not correct, since it supports prefixed strings. However, internally here we
+// explicitly use it when the strings are known to be unprefixed (for readability and avoid
+// confuction why we would not use our own `hexToBytes`)
```

Comment:
The first word here should be "confusion"?

Reply:
Yes, fixed